pkg/controller/pxc/users.go

```diff
@@ -0,0 +1,143 @@
+package pxc
+
+import (
+	"context"
+	"crypto/sha256"
+	"fmt"
+	"os"
+
+	api "github.com/percona/percona-xtradb-cluster-operator/pkg/apis/pxc/v1"
+	"github.com/percona/percona-xtradb-cluster-operator/pkg/pxc/users"
+	"github.com/pkg/errors"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+func (r *ReconcilePerconaXtraDBCluster) reconcileUsers(cr *api.PerconaXtraDBCluster) error {
+	if cr.Status.Status != api.AppStateReady {
+		return nil
+	}
+
+	if len(cr.Spec.Users.Secrets) == 0 {
+		return nil
+	}
+	secretObj := corev1.Secret{}
```

Comment:
why it is needed? looks like don't use it at all

Reply:
removed https://github.com/percona/percona-xtradb-cluster-operator/pull/342/commits/74d8e58743302032ea789cd46dfb590f31384fef